### DISCUSSION TITLE
Option to Apply redirectUrl immediately

### DISCUSSION THREAD

**Discussion by snicoll** [lxndrblz/anatole#414] [Q&A]
Is there a way to avoid the [one sec delay](https://github.com/lxndrblz/anatole/wiki/📄-Content-Settings#setting-up-external-redirect-urls) when a `redirectUrl` is set? I am having mostly empty pages that are used to gather content I've posted elsewhere and going to a blank full detail followed by a redirect is not ideal.

**Comment by lxndrblz** [comment]
@snicoll You would need to modify this bit of the theme:

https://github.com/lxndrblz/anatole/blob/800b13d687b8df22e9050874366e5ce79e7b80bf/layouts/partials/head.html#L31-L33

Simply change the 1 to 0 and the redirect will be almost instantly.

**Comment by snicoll** [comment]
Thanks for the reply. Maybe it's too much but I wonder if `externalUrl` can be added to control the behaviour. If that is set, that would be an immediate forward, not even showing the page itself.

I also wonder what the rationale is behind `redirectUrl` actually showing the related page for a second? If the actual content is external (which I believe is when we should be using this), I'd argue we shouldn't be showing the "local" page at all. What do you think?
